src/lib.rs

```diff
@@ -1098,6 +1098,21 @@ mod test {
     }
 
     #[test]
+    #[should_panic(expected = "Unexpected value")]
```

Comment:
We are not expecting a panic due to unwrap() here. We are expecting a panic for not giving integer value. Should this change?